test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -73,6 +73,32 @@ Describe "Get-Content" -Tags "CI" {
     It "should throw 'PSNotSupportedException' when you set-content to an unsupported provider" -Skip:($IsLinux -Or $IsMacOS) {
         {get-content -path HKLM:\\software\\microsoft -ea stop} | Should Throw "IContentCmdletProvider interface is not implemented"
     }
+    It 'Verifies -Tail reports a TailNotSupported error for unsupported providers' {
+        {Get-Content -Path Variable:\PSHOME -Tail 1 -ErrorAction Stop} | ShouldBeErrorId 'TailNotSupported,Microsoft.PowerShell.Commands.GetContentCommand'
+    }
+    It 'Verifies using -Tail and -TotalCount together reports a TailAndHeadCannotCoexist error' {
+            { Get-Content -Path Variable:\PSHOME -Tail 1 -TotalCount 5 -ErrorAction Stop} | ShouldBeErrorId 'TailAndHeadCannotCoexist,Microsoft.PowerShell.Commands.GetContentCommand'
+    }
+    It 'Verifies -Tail with content that uses an uncommon encoding' {
+        $content = @"
+one
+two
+foo
+bar
+baz
+"@
+        $expected = 'foo'
+        $tailCount = 3
+
+        $testPath = Join-Path -Path $TestDrive -ChildPath 'TailWithEncoding.txt'
+        $content | Set-Content -Path $testPath -Encoding BigEndianUnicode
+        $expected = 'foo'
+
+        $actual = Get-Content -Path $testPath -Tail $tailCount -Encoding BigEndianUnicode
+        ,$actual | Should BeOfType "System.Object[]"
```

Comment:
I don't see value in this check

Reply:
There are no tests that use a non-default encoding; other than byte. I can remove it but I see it as a gap.

Reply:
Since you use the comma in front, it will be an array so this check will always succeed.

Reply:
fixed